gometrics_test.go

```diff
@@ -33,7 +33,7 @@ func TestGoMetrics(t *testing.T) {
 		So(err, ShouldBeNil)
 		So(datapoints, ShouldNotBeNil)
 
-		So(len(datapoints), ShouldEqual, 29) // should be 29 because num_cgo_call should be 0 and ignored
+		So(len(datapoints), ShouldBeGreaterThan, 0)
```

Comment:
This seems to make the test more vague... Did something change to make it less predictable?